base/applications/notepad/dialog.c

```diff
@@ -820,7 +820,7 @@ VOID DIALOG_SearchNext(BOOL bDown)
     else
         Globals.find.Flags &= ~FR_DOWN;
 
-    if (Globals.find.lpstrFindWhat != NULL)
+    if (Globals.find.lpstrFindWhat != NULL && *Globals.find.lpstrFindWhat)
```

Comment:
Hmm... instead of either an error box or a search window, I would prefer just a system beep.

Reply:
UPD: nevermind, I misread the PR title somehow.

Reply:
No, you want the search window. I pressed F3 because I want to search.